graylog2-web-interface/src/views/components/visualizations/GenericPlot.tsx

```diff
@@ -152,6 +153,11 @@ class GenericPlot extends React.Component<GenericPlotProps, State> {
 
   render() {
     const { chartData, layout, setChartColor, theme } = this.props;
+    const fontSettings = {
+      color: theme.colors.global.textDefault,
+      size: ROOT_FONT_SIZE * theme.fonts.size.small.replace(/rem|em/i, ''),
```

Comment:
Unfortunately we need to define the size as a number (in px without a unit). A cleaner solution is to provide font sizes in pixels in the theme, like we do it for `spacings`, in a follow-up PR.